server/src/main/java/org/apache/iotdb/db/metadata/mnode/config/IConfigMNode.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.metadata.mnode.config;
+
+import org.apache.iotdb.commons.schema.node.IMNode;
+
+public interface IConfigMNode extends IMNode<IConfigMNode> {
```

Comment:
Should this class placed in confignode package?

Reply:
All config node related code should be migrated together. It may include IConfigMNode, ConfigMTree, Traverser, Template....  Lets migrate them in another PR.